.github/workflow_scripts/env_setup.sh

```diff
@@ -62,7 +62,7 @@ function install_multimodal {
     # launch different process for each test to make sure memory is released
     python3 -m pip install --upgrade pytest-xdist
     python3 -m pip install --upgrade -e multimodal/[tests]
-    mim install mmcv-full --timeout 1000
+    mim install mmcv-full --timeout 60
```

Comment:
I think that is because the server mim using is not stable enough. When there is a connection issue, setting this number too low will cause a timeout error, but too high will cause the import error while building tutorial.

Reply:
+1 to consensus. I prefer setting it to a relative lower number is better. If we get timeout error, we just run the code/CI again.

Reply:
How about setting it to 120? May be 1000 is too large. We can adjust the number again when we see flaky CI.

Reply:
Reached an agreement to set this to 60 in offline meeting.